wftests/warrior_tests/testcases/rest_functional_tests/rest_variable_in_json_test.xml

```diff
@@ -1,233 +1,326 @@
-			<Arguments>
-				<argument name="system_name" value="http_bin_1"/>
-				<argument name="url" value="http://httpbin.org/post"/>
-				<argument name="data" value="userId=1;id=1;title=Changed Post;body=New Comment"/>
-				<argument name="json" value='tag=tag9'/>
-				<argument name="cookies" value="cookie=cookie_name"/>
-				<argument name="allow_redirects" value="yes"/>
-				<argument name="expected_response" value="200, 302, 404"/>
-			</Arguments>
-			<onError action="next"/>
-			<Description>This step tests the POST capability of REST</Description>
-			<Execute ExecType="Yes"/>
-			<context>positive</context>
-			<impact>impact</impact>
-		</step>
-		<step Driver="rest_driver" Keyword="perform_http_post" TS="1">
-			<Arguments>
-				<argument name="system_name" value="http_bin_1"/>
-				<argument name="url" value="http://httpbin.org/post"/>
-				<argument name="data" value="userId=1;id=1;title=Changed Post;body=New Comment"/>
-				<argument name="json" value='../../config_files/rest_functional_tests/json_data.json'/>
-				<argument name="cookies" value="cookie=cookie_name"/>
-				<argument name="allow_redirects" value="yes"/>
-				<argument name="expected_response" value="200, 302, 404"/>
-			</Arguments>
-			<onError action="next"/>
-			<Description>This step tests the POST capability of REST</Description>
-			<Execute ExecType="Yes"/>
-			<context>positive</context>
-			<impact>impact</impact>
-		</step>
-		<step Driver="rest_driver" Keyword="perform_http_post" TS="1">
-			<Arguments>
-				<argument name="system_name" value="http_bin_1"/>
-				<argument name="url" value="http://httpbin.org/post"/>
-				<argument name="data" value="userId=1;id=1;title=Changed Post;body=New Comment"/>
-				<argument name="json" value=''/>
-				<argument name="cookies" value="cookie=cookie_name"/>
-				<argument name="allow_redirects" value="yes"/>
-				<argument name="expected_response" value="200, 302, 404"/>
-			</Arguments>
-			<onError action="next"/>
-			<Description>This step tests the POST capability of REST</Description>
-			<Execute ExecType="Yes"/>
-			<context>positive</context>
-			<impact>impact</impact>
-		</step>
-		<step Driver="rest_driver" Keyword="perform_http_post" TS="1">
-			<Arguments>
-				<argument name="system_name" value="http_bin_1"/>
-				<argument name="url" value="http://httpbin.org/post"/>
-				<argument name="data" value="userId=1;id=1;title=Changed Post;body=New Comment"/>
-				<argument name="json" value='../../config_files/rest_functional_tests/json_data.json'/>
-				<argument name="cookies" value="cookie=cookie_name"/>
-				<argument name="allow_redirects" value="yes"/>
-				<argument name="expected_response" value="200, 302, 404"/>
-			</Arguments>
-			<onError action="next"/>
-			<Description>This step tests the POST capability of REST</Description>
-			<Execute ExecType="Yes"/>
-			<context>positive</context>
-			<impact>impact</impact>
-		</step>
-	</Steps>
-
-</Testcase>
\ No newline at end of file
+        <Details>
```

Comment:
Changing an existing testcase in our established regression tests is not recommended. Please create a brand new testcase providing coverage to the functionality added in your pull request and add that to the suite.